build.gradle

```diff
@@ -735,3 +736,88 @@ task distRpm(dependsOn: "jpackageImage", type: Rpm) {
 	prefix '/opt' // Relocations: /opt
 	arch = 'x86_64'
 }
+
+task dockerPullOracleLinux8(type: DockerPullImage) {
+	group = "openjdk17-oraclelinux8 container"
```

Comment:
Maybe slighy shorter Gradle group name. The primary objective of this container is to build an ANET RPM which is RedHat 8 compliant.  Maybe "RH8 RPM build container"?